x-pack/plugins/search_inference_endpoints/common/translations.ts

```diff
@@ -21,29 +21,23 @@ export const CANCEL = i18n.translate('xpack.searchInferenceEndpoints.cancel', {
 export const MANAGE_INFERENCE_ENDPOINTS_LABEL = i18n.translate(
   'xpack.searchInferenceEndpoints.allInferenceEndpoints.description',
   {
-    defaultMessage: 'Manage your inference endpoints.',
-  }
-);
-
-export const ADD_ENDPOINT_LABEL = i18n.translate(
-  'xpack.searchInferenceEndpoints.newInferenceEndpointButtonLabel',
-  {
-    defaultMessage: 'Add endpoint',
+    defaultMessage:
+      'Inference endpoints streamline the deployment and management of machine learning models in Elasticsearch. Set up tasks such as text embedding, completions, and reranking using simple, unique endpoints.',
   }
 );
 
 export const CREATE_FIRST_INFERENCE_ENDPOINT_DESCRIPTION = i18n.translate(
   'xpack.searchInferenceEndpoints.addEmptyPrompt.createFirstInferenceEndpointDescription',
   {
     defaultMessage:
-      'Connect to your third-party model provider to create an inference endpoint for semantic search.',
+      "Inference endpoints streamline the deployment and management of machine learning models in Elasticsearch. Set up tasks such as text embedding, completions, and reranking using simple, unique endpoints. Use Elastic's built-in models, connect to third-party services via API, or integrate models uploaded with Python.",
   }
 );
 
 export const START_WITH_PREPARED_ENDPOINTS_LABEL = i18n.translate(
   'xpack.searchInferenceEndpoints.addEmptyPrompt.startWithPreparedEndpointsLabel',
   {
-    defaultMessage: 'Get started quickly with our prepared endpoints:',
+    defaultMessage: 'Get started quickly with our prepared built-in ML models:',
```

Comment:
cc: @leemthompo copy

Reply:
Updated based on this [suggestion](https://github.com/elastic/kibana/pull/188783/commits/c7499feaef726ef9257a156db93382495e1701a0).